cocos/core/game.ts

```diff
@@ -377,17 +377,14 @@ export class Game extends EventTarget {
 
     public _intervalId: number | null = null; // interval target of main
 
-    public _lastTime: Date | null = null;
-    public _frameTime: number | null = null;
+    public declare _lastTime: Date;
+    public declare _frameTime: number;
```

Comment:
This confuse me why `_lastTime` and `_frameTime` can be null. Please confirm this. @pandamicro